### PR TITLE
21 confirm role access

### PR DESCRIPTION
**Pull request by adberglund**
I expect verify_account_access to change immediately following this with #22

### CODE REVIEW

File: cloudigrade/util/aws.py [hunk 1]
```diff
@@ -49,7 +50,11 @@ def get_credentials_for_arn(arn):
 
     """
     sts = boto3.client('sts')
-    session_name = 'temp-session'  # TODO Does this need to be a funciton arg?
```
Comment (adberglund) [inline cloudigrade/util/aws.py]: I didn't meant to wipe your TODO, and will add it back in if wanted. My impression is that the session name matters very little. I could be very, very wrong.

File: cloudigrade/account/management/commands/add_account.py [hunk 2]
```diff
@@ -29,9 +29,13 @@ def handle(self, *args, **options):
...
         account_id = aws.extract_account_id_from_arn(arn)
 
-        # TODO AWS account permissions verification goes here.
-        found_instances = aws.get_running_instances(arn)
-        Account(account_arn=options['arn'], account_id=account_id).save()
-
-        self.stdout.write(self.style.SUCCESS(_('ARN Info Stored')))
-        self.stdout.write(_(f'Instances found include: {found_instances}'))
+        is_verified = aws.verify_account_access(arn)
+        if is_verified:
```
Comment (ilyavw) [inline cloudigrade/account/management/commands/add_account.py]: why not just `if aws.verify_account_access(arn)`, it doesn't look like we use `is_verified` anywhere else.

File: cloudigrade/util/aws.py (hunk 1 above)
Comment (infinitewarp) [reply cloudigrade/util/aws.py]: I'm okay with this! 👍

**Comment by infinitewarp** [comment]
Since you have a merge commit in the history here, please be sure to use "**Squash and merge**" when closing this.

**Comment by ilyavw** [comment]
Or to add to what Brad said above, squash it out manually if you'd like to preserve some other commits.

**Pull request merged**